underactuated/exercises/acrobot/test_multiple_pendulum_urdf.py

```diff
@@ -0,0 +1,179 @@
+import unittest
+import timeout_decorator
+from gradescope_utils.autograder_utils.decorators import weight
+import numpy as np
+
+class TestMultiplePendulumURDF(unittest.TestCase):
```

Comment:
our plan is to replace the existing problem, right?

Reply:
yes, replacing 3.2 in the textbook